ai-game/leaderboard.txt

```diff
@@ -0,0 +1 @@
+{"daan": "999", "ruben": "998", "floris": "997", "shaun": "63", "charlie": "42", "ducky": "33", "pieter": "31", "alex": "16", "duck": "14", "anonymous": "0"}
```

Comment:
JSON format in txt file?

Reply:
het zou misschien fouten veroorzaken in de code. Het txt format moet gehouden worden voor nu